desktop/cmp/input/CodeInput.js

```diff
@@ -426,18 +434,20 @@ class Model extends HoistInputModel {
 
 const cmp = hoistCmp.factory(
     ({model, className, ...props}, ref) => {
-        const childProps = {
+        return box({
             width: 300,
             height: 100,
             ...getLayoutProps(props),
-            className,
-            ref,
-            model
-        };
-
-        return modalSupport({
-            model: model.modalSupportModel,
-            item: inputCmp(childProps)
+            item: modalSupport({
+                model: model.modalSupportModel,
+                item: inputCmp({
+                    width: '100%',
+                    height: '100%',
+                    className,
```

Comment:
Do we still want the `className` to go here? I think typically we want the `className` on the outermost container, so devs always have a styling 'hook' into the component

Reply:
That makes sense.  I guess I was thinking that moving the `className` one component up in the hierarchy might be more disruptive to any custom styling devs were applying?  But I agree with you, better to have it applied to the outermost container- will make that change and test the `refresh()` method you mentioned above

Reply:
Sorry - upon taking a closer look at it, I realised this means our styling does not get applied **at all** in modal mode! Apologies for the mislead there.

I'll revert this change, and instead add a *new* className to the outer component.

Reply:
I had this thought too, but I think the styling does still get applied.  In modal mode, the `inputCmp` gets rendered inside a blueprint `Dialog` inside `box`.  So styling applied to `box` still would get applied to `inputCmp` I think, right?

Reply:
No, because the Dialog is shown in a portal nested at the bottom of `<body>`. So any inherited styles no longer apply.  You can see the difference if you compare e.g. the font size in toolbox-dev against this branch (prior to my latest push)

Reply:
Ah got it! That makes sense, thanks for clarifying